script/OWNERS

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
This OWNERS file is primarily added as a demonstration for how authors of specific scripts can add themselves on a per-file basis.